### PR TITLE
Validate SOBER

### PR DESCRIPTION
**Pull request by kissenry**
# Validate [SOBER](https://solscan.io/token/2ZE6hSL36e44wP168YMnxrbi1CSCFuD2BJm7NoNHfsmN)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/solabrador/status/1728802982416797790
- Coingecko/ CMC URL (If available): In progress

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by bossdenxy** [comment]
Awesome project 
Would be nice to have you on $Jup

**Comment by BrandonL5150** [comment]
We need $SOBER PLEASEEEEEEE

**Comment by Biboombi** [comment]
I want $SOBER

**Comment by jod1s** [comment]
WE NEED $SOBER PLS

**Comment by pranabnair** [comment]
Verify please

**Comment by OrionJelly** [comment]
$SOBER

**Comment by KujaSOL** [comment]
$SOBER needs to be verified

**Comment by was0107** [comment]
Let's get $SOBER verified

**Comment by ashu240699** [comment]
Please validate $SOBER we want it we need it we demand it

**Comment by maul37** [comment]
Let's Verified $SOBER  🔥

**Comment by tamsabogdan** [comment]
We need $Sober 🔥

**Comment by liuxiongfeng** [comment]
$SOBER plz

**Comment by Solprofessor565** [comment]
GIB VALIDATION ALREADY

**Comment by RogerKrog** [comment]
Let get $SOBER Approved! We need this!

**Review by velhusteth — COMMENTED** [review]
LFG 🔥 $SOBER verified !

**Comment by Vgr85** [comment]
$SOBER to get approved so it speeds up the process.

**Comment by guskala** [comment]
Please we need verify this token solabrador!

**Comment by guskala** [comment]
Please friends, we need verify token $sober of solobrador😎👌

**Comment by kenhalll** [comment]
I can confirm that the provided information is valid and all posted info is correct.

**Comment by dodge75** [comment]
I can confirm that the provided information is valid and all posted info is correct. $SOBER

**Comment by broadby** [comment]
$SOBER

**Comment by dcjona** [comment]
I can confirm that the provided information is valid and all posted info is correct.

**Comment by fiesty1063** [comment]
$SOBER needs verification ASAP. I can confirm that the provided information is valid and all posted info is correct.

**Comment by ImaDefender** [comment]
We need $SOBER to be verified, please.

I can confirm that the provided information is valid and all posted info is correct.

**Comment by ANIMAL6179** [comment]
I can confirm that the provided information is valid and all posted info is correct. 💪

**Comment by Gvlahoss** [comment]
I want $SOBER

**Comment by Oladare** [comment]
I can confirm that the provided information is valid and all posted info is correct.

**Comment by Oladare** [comment]
Sober is the shin on Solana pls we need it on Jupiter

**Comment by ppgeekle** [comment]
Please validate $SOBER, 
I've checked the notes and can confirm that the provided information is valid and all posted info is correct.

**Comment by rijal12fe** [comment]
Good project

**Comment by Caliente420** [comment]
I can confirm that the provided information is valid and all posted info is correct. $SOBER

**Comment by dupman01** [comment]
I can confirm that the provided information is valid and all posted info is correct.

**Review by Nelos88 — COMMENTED** [review]
I can confirm that the provided information is valid and all posted info is correct

**Comment by georgeSOL787** [comment]
I can confirm that the provided information is valid and all posted info is correct.

**Comment by georgeSOL787** [comment]
Bets project on Sol $SOBER

**Comment by sezerertugrul** [comment]
Awesome project
Would be nice to have you on $Jup

Sağlanan bilgilerin geçerli olduğunu ve gönderilen tüm bilgilerin doğru olduğunu onaylayabilirim. 💪

**Comment by gurcanmustafa** [comment]
I can confirm that the provided information is valid and all posted info is correct. Thanks

**Review by TonyTwoSmokes — COMMENTED** [review]
Seems legit LFG!

**Review by meijiao — DISMISSED** [review]

**Comment by 3ohdi** [comment]
I can confirm that the provided information is valid and all posted info is correct.

Don’t worry, Be Happy, Get $Sober!

**Comment by Wilkie93mdy** [comment]
Great 👍

**Comment by CryptoBingo101** [comment]
Please could you validate $Sober, going to do big things, majority of holders will not paperhand this token it’s got huge potential!
Thank you, all info presented is correct

**Comment by Prav33NN** [comment]
Validate $SOBER

**Comment by Prav33NN** [comment]
I can confirm that the provided information is valid and all posted info is correct.

**Comment by fomoshi** [comment]
I can confirm that the provided information is valid and all posted info is correct.

**Comment by YesineaSol** [comment]
I can confirm that the provided information is valid and all posted info is correct.

**Comment by kojilbae** [comment]
Nice $SOBER

**Comment by Cryptomoto11** [comment]
Please Approved $SOBER i want this to list in Jupiter

**Comment by maxkat** [comment]
I can confirm that the provided information is valid and all posted info is correct.

**Comment by Lord-Satchel** [comment]
LFG SOBER

**Comment by metawolf33** [comment]
Validate $SOBER. Verify $SOBER. MAKE IT HAPPEN. We are ready for it!

**Comment by metawolf33** [comment]
I can confirm that the provided information is valid and all posted info is correct.**

File: validated-tokens.csv [hunk 1]
```diff
@@ -817,3 +817,4 @@ Honeyland,HXD,3dgCCb15HMQSA4Pn3Tfii5vRk7aRqTH95LJjxzsG2Mug,9,https://arweave.net
 RIBBIT,RBT,65nTNuJGHme4PQvKQyJykKp1bJAkK4A8Q66sd2yBWugf,6,https://bafkreia3t3g3ecvvq7i5mgivvelysuhn5s6hzrulxmbkwgj5tgpsjtufge.ipfs.nftstorage.link,true
 Rocky,ROCKY,DhTNFBp4NjaxWqf6LwG57GrJQZfXrso1qK9FcMZt9sv7,9,https://imageupload.io/ib/B1wtisSaYlCUmwy_1700434086.jpg,true
 Ponk,PONK,HeqCcMjmuV5s25J49YiJyT6bD5qWLkP88YPajBySniaV,5,https://bafkreieyllpkega3ytsmq5gdwx2p44roka5pnlclw2an4xivqmbq7prmge.ipfs.nftstorage.link/,true
+Solabrador,SOBER,2ZE6hSL36e44wP168YMnxrbi1CSCFuD2BJm7NoNHfsmN,9,https://node2.irys.xyz/nMnAaHAJDeDTnGC9mJD9mXPbOQW9aBqpIXHZYftsgeY
```
Comment (Timoon21) [inline validated-tokens.csv:820]:
```suggestion
Solabrador,SOBER,2ZE6hSL36e44wP168YMnxrbi1CSCFuD2BJm7NoNHfsmN,9,https://node2.irys.xyz/5xOYKlM26hYXX5wkZoQhc1yw0Bd0Xzoh_SU52TakOZQ,true
```

**Comment by Timoon21** [comment]
@kissenry can you check and resolve the message error for merging : The base branch requires all commits to be signed. [Learn more about signing commits.](https://docs.github.com/articles/about-gpg/)

**Comment by sidquest2014** [comment]
I can confirm that the provided information is valid and all posted info is correct.

**Comment by kissenry** [comment]
> @kissenry can you check and resolve the message error for merging : The base branch requires all commits to be signed. [Learn more about signing commits.](https://docs.github.com/articles/about-gpg/)

Thanks for heads up - fixed.

**Comment by Kiron030** [comment]
I can confirm that the provided information is valide and the information is correct.

**Comment by Mansoor045** [comment]
I can confirm that the provided information is valid and all posted info is correct.

**Pull request merged**